pagseguro/__init__.py

```diff
@@ -112,7 +113,7 @@ class PagSeguro(object):
     NONE = 3
 
     def __init__(self, email, token, data=None, config=None):
-        self.config = config or Config()
+        self.config = isinstance(config, AbstractConfig) or Config()
```

Comment:
Nice, but failed in PEP8 check because it is more than 79 chars... 

```python
asser_error_msg = "Config should be instance of AbstractConfig"
assert isinstance(self.config, AbstractConfig), assert_error_msg
```